### PR TITLE
feat(nimbus): Add is_early_startup to NimbusVersionedSchema

### LINKED ISSUE #10041: Record isEarlyStartup status of desktop features in the database
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4197)

### PR DESCRIPTION
**Pull request by brennie**
Because:

- we want to warn users about writing large prefs on Firefox desktop; and
- Firefox Desktop will write values to prefs if features are marked `isEarlyStartup: true` in their feature config

this commit:

- adds the `is_early_startup` field to `NimbusVersionedSchema`; and
- updates the `load_feature_configs` command to update this information on new and existing features.

Fixes #10041

### CODE REVIEW

**Comment by brennie** [comment]
> No test updates?

I knew I was forgetting something

**Review by eliserichards — APPROVED** [review]
r+ when tests are added 👍

File: experimenter/experimenter/features/tests/test_load_feature_configs.py [hunk 1]
```diff
@@ -130,6 +131,7 @@ def test_updates_existing_feature_configs(self):
                 "additionalProperties": False,
             },
         )
+        self.assertTrue(schema.is_early_startup)
```
Comment (jaredlockhart) [inline experimenter/experimenter/features/tests/test_load_feature_configs.py:134]: 👍 🙏

**Review by jaredlockhart — APPROVED** [review]

File: experimenter/experimenter/experiments/tests/factories.py [hunk 2]
```diff
@@ -114,6 +114,7 @@ class NimbusVersionedSchemaFactory(factory.django.DjangoModelFactory):
     )
     schema = factory.LazyAttribute(lambda o: FAKER_JSON_SCHEMA)
     sets_prefs = factory.LazyAttribute(lambda o: [])
+    is_early_startup = factory.LazyAttribute(lambda o: False)
```
Comment (jaredlockhart) [inline experimenter/experimenter/experiments/tests/factories.py:117]: Oh you can just say 

```py
is_early_startup = False
```

for static values like that, you should only need the lambda for computed values?

File: experimenter/experimenter/features/management/commands/load_feature_configs.py [hunk 3]
```diff
@@ -153,18 +153,19 @@ def handle(self, *args, **options):
...
+            schema = schemas.get((feature_config.id, feature_version_id))
             if schema is None:
                 created = True
                 schema = NimbusVersionedSchema(
                     feature_config=feature_config,
                     version=feature_version,
+                    is_early_startup=feature.model.is_early_startup or False,
                 )
+            if schema.is_early_startup is None:
+                import ipdb; ipdb.set_trace()
```
Comment (jaredlockhart) [inline experimenter/experimenter/features/management/commands/load_feature_configs.py:168]: Oops

**Pull request merged**